<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemChooser</class>
 <widget class="QWidget" name="SystemChooser">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select a coordinate system</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QRadioButton" name="rectBtn">
     <property name="text">
      <string>Rectangle coordinate system</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radialBtn">
     <property name="text">
      <string>Radial coordinate system</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="rectFrm">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Center of coordinates:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="centerBox"/>
      </item>
      <item>
       <widget class="QGroupBox" name="yBox">
        <property name="title">
         <string>Y axis orientation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="toDownBtn">
           <property name="text">
            <string>To down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="toUpBtn">
           <property name="text">
            <string>To up</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="shiftXLbl">
          <property name="text">
           <string>X shift</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="shiftYLbl">
          <property name="text">
           <string>Y shift</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="shiftXSpin">
          <property name="minimum">
           <number>-10000</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="shiftYSpin">
          <property name="minimum">
           <number>-10000</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
